docs/storefront/headless/routes.mdx

```diff
@@ -0,0 +1,204 @@
+# Routes with the GraphQL Storefront API
+
+Routes define specific URL paths for fetching or updating data. The BigCommerce GraphQL Storefront API allows you to retrieve routes for each storefront channel, making it easier to build headless storefront applications using BigCommerce's built-in content. 
+
+When querying a `route`, the response may include the following node types:
+
+* Product
+* Category
+* Brand
+* NormalPage
+* ContactPage
+* RawHtmlPage
+* Blog
+* BlogPost
+
+## Enhanced redirect support
```

Comment:
I would just call this "Redirect support" since we have not previously documented this redirect behavior so adding the term "Enhanced" doesn't include a point of reference by which it is an enhancement.